scripts/install/run.sh

```diff
@@ -46,17 +46,26 @@ fi
 
 CLONE_DIR="${BASE_DIR}/${CLONE_DIR}"
 INSTALLATION_DIR="${BASE_DIR}/${INSTALLATION_DIR}"
+SKIP_SANITY=false
```

Comment:
maybe these new settings should be in `config.default.sh`.

Reply:
yes much nicer! :)